.github/workflows/test.yml

```diff
@@ -33,31 +33,10 @@ jobs:
       - name: Install dependencies
         run: |
               python -m pip install --upgrade pip
-              pip install setuptools tox tox-gh-actions wheel
-
-      - name: Build with Werror and Wall
```

Comment:
I noticed we have this in the system lib build but not this one.

Reply:
Yeah, that was intentional. The purpose is to build the library with `-Werror -Wall -Wextra`. Given that we can't change `libmaxminddb`, it seemed better to only fail on warnings from the extension. That said, I see that I didn't quite set it up properly in the other workflow. I'll fix that.